### DISCUSSION TITLE
Your account is currently locked from purchases. Please update your payment information

### DISCUSSION THREAD

**Discussion by BorisTr32** [community/community#126300] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm seeing an error saying Your account is currently locked from purchases. Please update your payment information. after I submit the GitHub Copilot payment setup with a bank cc .

**Comment by Akash1134** [comment]
Hi @BorisTr32 ,

I recommend you to contact GitHub Support directly for this issue. They are equipped to handle account and billing issues and can assist you further. You can reach them through [GitHub Support](https://support.github.com/contact).
